.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -12,6 +12,9 @@ jobs:
       - bash: echo "##vso[task.prependpath]$CONDA/bin"
         displayName: Add conda to PATH
 
+      - bash: sudo apt-get install libxerces-c
+        displayName: Add conda to PATH
```

Comment:
Dependencies should be installed with mamba or added in the `environment-dev.yml` so be sure they're built with a compatible runtime., instead of relying on the build machine.

Reply:
It turns out my attempt does nothing.